rcl/src/rcl/arguments.c

```diff
@@ -1444,9 +1653,32 @@ _rcl_parse_log_level(
     return RCL_RET_OK;
   }
   RCL_SET_ERROR_MSG("Argument does not use a valid severity level");
-  return RCL_RET_INVALID_LOG_LEVEL;
+  return RCL_RET_ERROR;
 }
 
+rcl_ret_t
+_rcl_parse_log_level_rule(
```

Comment:
Might be worth adding a TODO or \deprecated marker to remove this function in F-turtle (same for others).

Reply:
I've added so in each forward declaration, but didn't duplicate it here. Would you do that twice?

Reply:
It's fine, I missed it.